Dockerfile

```diff
@@ -2,6 +2,8 @@
 
 FROM mozillamarketplace/centos-mysql-mkt:0.2
 RUN yum install -y supervisor bash-completion && yum clean all
+# This is for the cryptography module:
+RUN yum install -y gcc libffi-devel python-devel openssl-devel && yum clean all
```

Comment:
I think you'll find gcc is already installed in a previous container, I think python-devel is also there. Not sure about the others though.

https://github.com/mozilla/marketplace-env/blob/master/docker/base-images/python27/Dockerfile#L5